src/Wizard/Platform/Readline/ReadlinePlatform.ts

```diff
@@ -0,0 +1,147 @@
+import { Platform } from '../Platform';
+import { inject, injectable } from 'tsyringe';
+import { Helpers } from 'src/Utils';
+import { Tokens } from 'src/Wizard/Tokens';
+import { TestType } from 'src/Wizard/Models';
+import { ConnectivityService } from 'src/Wizard/Services/ConnectivityService';
```

Comment:
Wrong paths, please fix them

Reply:
oh, yes, thanks. I had a problem with circle dependencies and made a test.